src/client.js

```diff
@@ -149,13 +149,24 @@ function getLoop54Client (endpoint, userId, apiKey, customHeaders) {
          */
         getRelatedEntities: function (entity) {
 
-            var args = core.getOptionsAndCallback(arguments, 1);
+            var args = core.getOptionsAndCallback(arguments, 1, 4);
             if (args.error) {
                 return core.returnError(args.error,args.callback);
             }
 
             var options = args.options ? args.options : {};
             var callback = args.callback ? args.callback : null;
+            
+            // read and validate relKind
+            var relKind;
+            for (var i = 1; i < arguments.length; i++) {
+                if (typeof(arguments[i]) === "string")
+                    relKind = arguments[i];
+            }
+            
+            if ((typeof(relKind) != "undefined") &&
```

Comment:
I think this is a bit easier to read `!(relKind === undefined || relKind === 'similar' || relKind === 'complementary')`